Dockerfile

```diff
@@ -6,7 +6,8 @@ RUN apk add --no-cache \
     && apk add --no-cache openvpn openssh \
     && apk add --no-cache py3-pip \
     && apk add --no-cache bind-tools curl \
-    && pip --no-cache-dir install pproxy supervisor
+    && apk add --no-cache supervisor \
+    && apk add --no-cache py3-pproxy \
```

Comment:
docker build issues a warning due to the last `\` 
WARN: NoEmptyContinuation: Empty continuation line (line 16)

Also, I can't find py3-pproxy in [alpine repository](https://pkgs.alpinelinux.org/packages?name=py3-pproxy&branch=edge&repo=testing)?

Reply:
Just merged #20 that probably fixes this. I'm able to build with master branch now.